test/models/metrics/algorithms_test.rb

```diff
@@ -38,6 +39,11 @@ class Algorithms < ActiveSupport::TestCase
     assert aggregate.pmid == 1
   end
 
+  test 'journal exact counts are included in monthly aggregation' do
+    aggregate = Metrics::Algorithms.new.generate(DateTime.now)
+    assert aggregate.journal_exact == 1
```

Comment:
I worry a bit that this type of testing will lead to similar issues as we've seen in ETD (i.e.,adding a fixture breaks a bunch of tests). Not requesting a change, partly because I don't have any good suggestions and partly because I'd love to get this ticket to done.

Reply:
Yeah, you are definitely correct. As more fixtures are added, this test could fail.

I don't have a great idea that doesn't involve just not using fixtures or reimplementing the algorithm that calculates matches in the test itself... neither of which are really acceptable.

Thanks for pointing this out as I do think it's important for us to acknowledge when these "future" problems are being introduced.